plugin/pkg/scheduler/factory/factory.go

```diff
@@ -436,6 +454,21 @@ func (c *configFactory) invalidatePredicatesForPvc(pvc *v1.PersistentVolumeClaim
 	}
 }
 
+func (c *configFactory) invalidatePredicatesForPvcUpdate(old, new *v1.PersistentVolumeClaim) {
```

Comment:
Ideally, we would like to have integration tests for these changes, but given that this PR is already too large and we are close to the code freeze, you may want to add the tests in a separate PR.

Reply:
Will open a bug to add integration tests for predicate invalidation.